sdk/aqueduct/utils/local_data.py

```diff
@@ -0,0 +1,71 @@
+import os
+from typing import Optional
+
+from aqueduct.constants.enums import ArtifactType, LocalDataTableFormat
+from aqueduct.models.local_data import LocalData
+
+from ..error import InvalidUserArgumentException
+
+# The current local data maximum is temporarily set to 1MB.
+MAXIMUM_LOCAL_DATA_SIZE = 1000000
+
+
+def Local_Data(
+    path: str,
+    artifact_type: ArtifactType,
+    format: Optional[str] = None,
+) -> LocalData:
+    """Identify the local data which can be passed in as a parameter.
+
+    Args:
+        path:
+            The path to the data.
+        artifact_type:
+                The expected type of the data. The `ArtifactType` class in `enums.py` contains all
```

Comment:
This is a user-facing function. Let's not point them to code. Let's just document the supported possibilities right now: ArtifactType.TABLE, ArtifactType.IMAGE